recipes/basemap/basemap-1.0.8dev-datadir.patch

```diff
@@ -0,0 +1,29 @@
+--- lib/mpl_toolkits/basemap/__init__.py	2016-03-23 10:25:48.150934435 -0300
++++ lib/mpl_toolkits/basemap/__init__.py	2016-03-23 11:10:03.794816035 -0300
+@@ -45,7 +45,10 @@
+     if not os.path.isdir(basemap_datadir):
+         raise RuntimeError('Path in environment BASEMAPDATA not a directory')
+ else:
+-    basemap_datadir = os.sep.join([os.path.dirname(__file__), 'data'])
++  if os.name == 'nt':
++    basemap_datadir = os.path.join(sys.prefix, 'Library', 'share', 'basemap')
++  else:
++    basemap_datadir = os.path.join(sys.prefix, 'share', 'basemap')
+ 
+ __version__ = '1.0.8'
+ 
+@@ -1206,10 +1209,10 @@
+         """
+         msg = dedent("""
+         Unable to open boundary dataset file. Only the 'crude', 'low',
+-        'intermediate' and 'high' resolution datasets are installed by default.
+-        If you are requesting a 'full' resolution dataset, you may need to
+-        download and install those files separately
+-        (see the basemap README for details).""")
++        and 'intermediate' resolution datasets are installed by default.
++        If you are requesting a 'high' or 'full' resolution dataset, you may need to
++        download and install those files separately with
++        `conda install -c conda-forge basemap-data-hires`.""")
```

Comment:
Make sure that people trying to use `h` and `f` will know what to do once basemap fails.